bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/impl/SingleProbeStrategyParams.scala

```diff
@@ -104,4 +111,48 @@ case class PwfsParams(site: Site, guideProbe: PwfsGuideProbe) extends SingleProb
   override def validator(ctx: ObsContext): GuideStarValidator =
     vignettingProofPatrolField(ctx).validator(ctx)
   }
-}
\ No newline at end of file
+}
+
+case object Pwfs1NGS2Params extends SingleProbeStrategyParams {
+  override val guideProbe: PwfsGuideProbe = PwfsGuideProbe.pwfs1
+  override val site: Site                 = Site.GS
+  override def stepSize: Angle            = Angle.fromDegrees(360)
+  override val catalogName: CatalogName   = PPMXL
```

Comment:
Always `PPMXL` regardless of what what the manual search catalog selection may be.